dependencies.gradle

```diff
@@ -7,17 +7,17 @@ ext.versions = [
         compileSdk                   : 23,
         buildTools                   : '26.0.2',
 
-        androidGradlePlugin          : '2.2.1',
-        retrolambdaGradlePlugin      : '3.2.5',
+        androidGradlePlugin          : '2.3.3',
+        retrolambdaGradlePlugin      : '3.7.0',
         lombokGradlePlugin           : '0.2.3.a2',
         paperworkGradlePlugin        : '1.2.7',
         jacocoGradlePlugin           : '0.7.6.201602180812',
         androidDevMetricsGradlePlugin: '0.4',
-        errorProneVersion            : '0.0.8',
+        errorProneGradlePlugin       : '0.0.13',
         errorProneCoreVersion        : '2.0.9',
```

Comment:
We could bump to 2.1.1 or even 2.1.2

Reply:
Yep, in a separate PR :) I'm afraid that it could show new errors/warnings and I'll have to fix them in unrelated PR